CONTRIBUTING.md

```diff
@@ -109,7 +112,7 @@ create_sfp(title = "Klassieke vegetatieopname in een proefvlak aan de hand van v
            date = "2016-07-19", 
            reviewers = "Hans Van Calster, Lieve Vriens, Jan Wouters, Wouter Van Gompel, Els Lommelen", 
            file_manager = "Hans Van Calster", 
-           revision = "1.1.0.9000",
+           version_number = "2020.00.dev",
```

Comment:
Good catch to update the argument name. However, here I'd suggest to _drop_ the argument here and use a more convenient default in `create_sfp()`. Now that is:

`version_number = "YYYY.##.dev"`

But probably something like `paste0(format(Sys.Date(), "%Y"), ".00.dev")` would be more convenient as a default?

- it would default to current year
- it would default to '00' (which should be the case when creating Rmd files for the first time)

Reply:
Good idea! Will change that here and in protocolhelper::create_sfp

Reply:
See 59e2aef

Reply:
and also inbo/protocolhelper@6a7090f